react-sanfona/build.boot

```diff
@@ -1,15 +1,17 @@
 (set-env!
   :resource-paths #{"resources"}
-  :dependencies '[[cljsjs/boot-cljsjs "0.9.0"  :scope "test"]
-                  [cljsjs/react       "15.3.0-0"]])
+  :dependencies '[[cljsjs/boot-cljsjs "0.10.0"  :scope "test"]
+                  [cljsjs/react       "16.3.0-0"]
+                  [cljsjs/react-dom   "16.3.0-0"]
+                  [cljsjs/react-dom-server "16.3.0-0"]])
```

Comment:
Probably no need to require dom-server package here, it is deprecated and the react-dom/server file is not part of react-dom package.